react-components/src/notifications/view/dialogs/DenyJoinRequestDetailsDialog.js

```diff
@@ -41,21 +41,21 @@ class DenyJoinRequestDetailsDialog extends Component {
                 </DialogTitle>
                 <DialogContent>
                     <DialogContentText>
-                        {getMessage("denyDetailsMessage", {
+                        <Typography>{getMessage("denyDetailsMessage", {
                             values: {
                                 team: teamName
                             }
-                        })}
+                        })}</Typography>
                         <p style={{marginTop: 10}}>
-                            <Typography subheading>{getMessage("denyAdminLabel")}</Typography>
+                            {getMessage("denyAdminLabel")}
                             <TextField
                                 InputLabelProps={{
                                     shrink: true,
+                                    readOnly: true
```

Comment:
My last comment recommended `InputProps={{ readOnly: true }}`, which is different than the `InputLabelProps` here.

I don't think this will disable user editing for the input.

Reply:
It did... Unless am missing something.

Reply:
So I checked the DOM.

With InputLabelProps
`<input aria-invalid="false" class="MuiInput-input-84" type="text" value="Nope. You are not cool.">`

With InputProps
`<input aria-invalid="false" class="MuiInput-input-84" readonly="" type="text" value="Nope. You are not cool.">`

Not sure why it not editable with InputLabelProps. Anyway I will go ahead and change it to use InputProps.